src/actions/breakpoints/addBreakpoint.js

```diff
@@ -29,7 +29,13 @@ import type { SourceLocation } from "../../types";
 import type { ThunkArgs } from "../types";
 import type { addBreakpointOptions } from "./";
 
-async function addBreakpointPromise(getState, client, sourceMaps, breakpoint) {
+async function addBreakpointPromise(
+  getState,
+  client,
+  sourceMaps,
+  breakpoint,
+  shiftKey = false
```

Comment:
See earlier comment - `shiftKey` parameter might be redundant if it's always the same value as `breakpoint.disabled`